### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by moath143**
Issue #10 
Please review the footer section code in footer branch

### CODE REVIEW

File: src/components/checkboxCatalog/CheckBoxCatalog.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <>
+      <FormControlLabel
+        control={
+          <Checkbox
+            sx={{
+              color: "#000",
+              "&.Mui-checked": {
+                color: "#D1094B",
+              },
+            }}
```
Comment (Mossa-99) [inline src/components/checkboxCatalog/CheckBoxCatalog.tsx:18]: inline styles

File: src/components/footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+const Footer: React.FC = () => {
+  return (
+    <FooterSection>
+      <Container>
+        <Grid
+          sx={{
+            display: "flex",
+            justifyContent: "space-between",
+            alignItems: "start",
+            padding: "0",
```
Comment (Mossa-99) [inline src/components/footer/Footer.tsx:16]: inline styles

File: src/components/footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+              <li>legal policies</li>
+            </FooterList>
+          </Grid>
+          <Grid item xs={4}>
+            <Typography
+              sx={{
+                fontWeight: "400",
+                fontSize: "24px",
+                lineHeight: "24px",
+                textTransform: "uppercase",
```
Comment (Mossa-99) [inline src/components/footer/Footer.tsx:68]: Please use styled components

File: src/pages/catalog/Catalog.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+                ),
+              }}
+            />
+            <Typography sx={{ margin: "30px 0 10px 0" }} variant="h5">
+              TOPS
+            </Typography>
+            <ImagesTops>
+              <ImagesTopsCatalog />
+            </ImagesTops>
+            <ImagesShirts style={{ margin: "30px 0 10px 0" }}>
```
Comment (Mossa-99) [inline src/pages/catalog/Catalog.tsx:69]: inline styles

**Review by Mossa-99 — CHANGES_REQUESTED** [review]

**Review by yasmeentayeh99 — APPROVED** [review]

**Pull request merged**